components/form/Footer.vue

```diff
@@ -14,6 +14,14 @@ export default {
     errors: {
       type:    Array,
       default: null,
+    },
+
+    /**
+     * A value to tell async button to use the enabled labels instead of the default create labels
+     */
+    useEnableLabelForCreate: {
```

Comment:
This should take an arbitrary mode to send to the AsyncButton instead of only working for this one special case..  like an optional `saveButtonMode` prop you can set to `enable`.

There's also not really any need for there to be 2 buttons and an if on each, since they're never both true.

This can just be like `<AsyncButton v-if="!isView" :mode="saveButtonMode || mode" @click="save" />`